tests/db_migrator_input/config_db/reclaiming-buffer-warmreboot-expected.json

```diff
@@ -2043,6 +2043,6 @@
         "admin_status": "up"
     },
     "VERSIONS|DATABASE": {
-        "VERSION": "version_3_0_5"
+        "VERSION": "version_3_0_3"
```

Comment:
Why is this change needed? Is this relevant to persistance loglevel design?

Reply:
Yes, relevant to persistance loglevel design.

Reply:
I see. I am unable to understand, can you explain why tests for reclaiming-buffer need to be changed for persistence log-level design?

Reply:
This is to change it back. Refer [this comment](https://github.com/sonic-net/sonic-utilities/pull/2370#discussion_r992945987) for details